.github/workflows/test-ipynb.yml

```diff
@@ -0,0 +1,42 @@
+name: Testing Jupyter Notebooks
+
+on:
+  push:
+    branches:
```

Comment:
As discussed, this should also include `on-dispatch` case

Reply:
Replaced the `push` with the `workflow_dispatch`.